tox.ini

```diff
@@ -148,6 +148,22 @@ commands=
 
   testinfra -n 4 --sudo -v --connection=ansible --ansible-inventory={changedir}/hosts-switch-to-containers {toxinidir}/tests/functional/tests
 
+[add-mons]
+commands=
+  ansible-playbook -vv -i {changedir}/hosts-2 --limit mon1 {toxinidir}/tests/functional/setup.yml
+  cp {toxinidir}/infrastructure-playbooks/add-mon.yml {toxinidir}/add-mon.yml
+  ansible-playbook -vv -i {changedir}/hosts-2 --limit mon1 {toxinidir}/add-mon.yml --extra-vars "\
```

Comment:
if you limit the run to the added monitor, other monitors won't get their ceph.conf updated correctly